Backend/DataAccessLayer/Repositories/UserManagementRepository.cs

```diff
@@ -67,5 +67,17 @@ public bool ExistingUser(DatabaseContext _db, string email)
             }
             return false;
         }
+
+        public bool IsManagerOf(DatabaseContext _db, User user, User subject)
+        {
+            if (subject == null || subject.ManagerId == null) return false;
+
+            if (subject.ManagerId == user.Id) return true;
+
+            // Check if we're a manager of the subjects manager
+            var managerOfSubject = GetUser(_db, (Guid) subject.ManagerId);
+            if (managerOfSubject == null) return false;

```

Comment:
No need for this line if you check for null above.

Reply:
True, the next recursive call will always check this. I'll remove it tomorrow unless you want to make that as a quick fix.